index.js

```diff
@@ -1,42 +1,58 @@
 const Discord = require('discord.js');
+const {Uni} = require('./uniBot');
 const {prefix, token, giphyToken, ytToken} = require('./config');
 const client = new Discord.Client();
-const quotes = require('./quotes');
+
 const GphApiClient = require('giphy-js-sdk-core');
 giphy = GphApiClient(giphyToken);
 const fetch = require('node-fetch');
 
-client.once('ready', () => {console.log("Unicorn trotting!")});
+
+client.once('ready', () => {
+    console.log("Unicorn trotting!")
+});
 
 //!uni
 client.on('message', message => {
-    if (message.content.startsWith(`${prefix}uni`)) {
-        let member = message.mentions.members.first() || "";
-        giphy.search('gifs', {"q": ["magical", "unicorn", "animated candies", "fairyland"]})
-            .then((response) => {
-                //getting one gif
-                let totalResponses = response.data.length;
-                let responseIndex = Math.floor((Math.random() * 10) + 1) % totalResponses;
-                let finalResponse = response.data[responseIndex];
-                //getting one quote
-                let totalQuotes = quotes.length;
-                let quotesIndex = Math.floor((Math.random() * 10) + 1) % totalQuotes;
-                let quote = quotes[quotesIndex];
-
-                message.channel.send(`${quote} ${member} :unicorn:`, {
-                    files: [finalResponse.images.fixed_height.url]
-                });
-            })
-            .catch(() => {
-                message.channel.send('Hiccup...')
-            });
+    try {
+        if (message.content.startsWith(`${prefix}uni`)) Uni(message);
+        else if (message.content === 'ping') message.reply('pong');
+        else if (message.content === '!flip' || message.content === '!f') message.reply(Math.round(Math.random()) === 1 ? ' ♕ Heads!' : ' ♘ Tails!');
+        else if (message.content.startsWith('!scrim')) scrim(message)
+    } catch {
+        //message.reply('Something broke, should have written tests.');
     }
 });
 
-//ping-pong
-client.on('message', msg => {
-    if (msg.content === 'ping') msg.reply('pong');
-});
+const scrim = async (message) => {
+    const CHANNEL_CHEESE_CAKE = '310195830290382848';
+    const channel = await client.channels.fetch(CHANNEL_CHEESE_CAKE);
+    const usersInChannel = [];
+    //console.log(channel.members);
+    for (let [sf, member] of channel.members) {
+        console.log(member);
+        const name = member.nickname ? member.nickname : member.user.username;
+        usersInChannel.push(name);
+    }
+    const allUsers = shuffle(usersInChannel);
+
+    let teamMessage = `> Team ${allUsers[0]}:\n`;
+    const half = allUsers.length / 2;
```

Comment:
this one should be a Math.floor(allUsers.length / 2)

Reply:
so we can support uneven teams?

Reply:
this worked nicely, also added a check to ensure there are 2 players